test/runner_shared.sh

```diff
@@ -38,7 +38,10 @@ shift
 # we use mkdir here because it is an atomic operation unlike existance of a lockfile
 # where creating and checking are 2 separate operations
 if mkdir ${TEST_OUTPUT_DIR}/.pg_init 2>/dev/null; then
-  ${PSQL} "$@" -U ${USER} -d postgres -v ECHO=none -c "ALTER USER ${TEST_ROLE_SUPERUSER} WITH SUPERUSER;" >/dev/null
+  ${PSQL} "$@" -U ${USER} -d postgres -v ECHO=none >/dev/null <<EOF
+GRANT ALL ON SCHEMA public TO PUBLIC;
```

Comment:
Shouldn't we do here the same you did in the `runner.sh`?

Reply:
It is not necessary here, so would like to re-visit this separately since it is not critical for this fix.